modules/ROOT/pages/dw-core-functions-distinctby.adoc

```diff
@@ -68,69 +73,71 @@ output application/json
 ----
 
 
-
 [[distinctby2]]
+== distinctBy(Null, (item: Nothing, index: Nothing) -> Any): Null
+
+Helper function that allows *distinctBy* to work with null values.
+
+
+[[distinctby3]]
 == distinctBy({ (K)?: V }, (value: V, key: K) -> Any): Object
 
-Returns an object with unlike key value pairs.
+Removes duplicate key-value pairs from an Object.
+
 
-The function (a lambda) is invoked with two parameters: `value` and `key`.
-If these parameters are not defined, the index is defined by default as `&#36;&#36;`
-and the value as `&#36;``.
+=== Parameters
 
-.Transform
-[source,dataweave,linenums]
+[%header, cols="1,3"]
+|===
+| Name   | Description
+| `object` | The object from which to remove the key-value pairs.
```

Comment:
How about The object whose key-value pairs are being removed or The object that contains key-value pairs to be removed.